src/m1_facilitator/repositories/AnchorRepository.ts

```diff
@@ -26,11 +26,11 @@ import assert = require('assert');
 /* eslint-disable class-methods-use-this */
 
 /**
- * An interface, that represents a row from an anchor table.
+ * An interface, that represents an anchor database model.
```

Comment:
Otherwise, we should make `get` and `save` functions static to eliminate linter error.
I definitely do not like to see them as a static (this happens because of sequelize).